src/App.js

```diff
@@ -1,23 +1,81 @@
-import logo from './logo.svg';
+import { useState, useEffect } from 'react';
 import './App.css';
+import Movie from './Movie';
+import MovieForm from './MovieForm';
+import MovieList from './MovieList';
+
 
 function App() {
+  const [allMovies, setAllMovies] = useState([]);
+  const [filteredMovies, setFilteredMovies] = useState('');
```

Comment:
This is absolutely a nitpick but I would suggest initializing filtered movies as an empty array. When you call `setFilteredMovies` you make it an array but the point of declaring an empty array initially is to let the coder who comes in after you know that this is going to hold an array eventually and it's just somewhat confusing. All that being said, it's a total nit pick and not that big of a deal.